### PR TITLE
Update RockLib.HealthChecks.AspNetCore dependency to RockLib.HealthChecks

### PR DESCRIPTION
**Pull request by cdraper033**
## Description
Updated RockLib.HealthChecks.AspNetCore to `3.0.0-alpha.2`
Updated RockLib.HealthChecks package reference to `2.0.1`

## Type of change:  2 

1. Non-functional change (e.g. documentation changes, removing unused `using` directives, renaming local variables, etc)
2. Bug fix (non-breaking change that fixes an issue)
3. New feature (non-breaking change that adds functionality)
4. Breaking change (fix or feature that could cause existing functionality to not work as expected)

## Checklist:

- Have you reviewed your own code? Do you understand every change?
- Are you following the [contributing guidelines](../blob/main/CONTRIBUTING.md)?
- Have you added tests that prove your fix is effective or that this feature works?
- New and existing unit tests pass locally with these changes?
- Have you made corresponding changes to the documentation?
- Will this change require an update to an example project? (if so, create an issue and link to it)

---

<sup>_[Reviewer guidelines](../blob/main/CONTRIBUTING.md#reviewing-changes)_</sup>

### CODE REVIEW

**Review by aatchison14 — APPROVED** [review]

File: RockLib.HealthChecks.AspNetCore/CHANGELOG.md [hunk 1]
```diff
@@ -5,29 +5,34 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 3.0.0-alpha.2 - 2023-02-28
+
+### Changed
+- Updated RockLib.HealthChecks package reference to `2.0.1`
+
 ## 3.0.0-alpha.1 - Not Yet Released
```
Comment (NeilSchiefer) [inline RockLib.HealthChecks.AspNetCore/CHANGELOG.md:13]: You could probably put the date on here since it was already released.

Comment (cdraper033) [reply RockLib.HealthChecks.AspNetCore/CHANGELOG.md:13]: Done

**Review by kristycurrier — CHANGES_REQUESTED** [review]
Hold until we fix the build issues

File: RockLib.HealthChecks.AspNetCore/CHANGELOG.md (hunk 1 above)
Comment (NeilSchiefer) [reply RockLib.HealthChecks.AspNetCore/CHANGELOG.md:13]: I don't see that this is updated.  For clarity I was talking about '3.0.0-alpha.1 - Not Yet Released'.

Comment (cdraper033) [reply RockLib.HealthChecks.AspNetCore/CHANGELOG.md:13]: Oof stupid Visual Studio showed an error message when I pushed it that I just noticed. But I'll push it once we get the build issues taken care of

**Pull request closed without merging**